### DISCUSSION TITLE
Make constraints lighten up / pop up and glow to visualize what is blocking

### DISCUSSION THREAD

**Discussion by NoteToSelfFindGoodNickname** [hlorus/CAD_Sketcher#286] [Feature Requests]
Hello!

Here is a video:

https://youtu.be/pYUU0z4dTXk

It shows how I try to move a point. I don't understand whay it's not working.
I must admit that I am really new to this, and perhaps with more experience, I could see it right away.

But if others have the same problem, even if they are already experienced, don't you think that it would both be cool and very informative if the contraints would lighten up to show that they are they cause of the blocking / not allowing my action to happen?

**Comment by hlorus** [comment]
Would be great indeed but i'm afraid the solver doesn't support reporting which exact constraints are limiting the degrees of freedom of a specific entity.
